docs: clarify login url usage (#9963)
vercel/turborepo: docs/repo-docs/reference/login.mdx

### --login \<url>

> Set the URL for login requests. This is only required for platforms implementing multi-tenancy.

Author: ```suggestion
Set the URL for login requests that should dynamically generate tokens.
```
